### PR TITLE
Update howlongtobeat extension

### PR DESCRIPTION
**Pull request by whitespacecode**
## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/how-long-to-beat/src/index.tsx [hunk 1]
```diff
@@ -53,6 +53,7 @@ function SearchListItem({ searchResult }: { searchResult: HowLongToBeatEntry })
 function useSearch() {
   const [state, setState] = useState<SearchState>({ results: [], isLoading: true });
 
+
```
Comment (Cronos87) [inline extensions/how-long-to-beat/src/index.tsx:56]: You can fix this by running version 2.7.1 of Prettier.

**Review by Cronos87 — APPROVED** [review]

**Comment by Cronos87** [comment]
Many thanks for fixing this! I had no time for it, and I'm happy someone fixed it! 🙏

File: extensions/how-long-to-beat/src/index.tsx (hunk 1 above)
Comment (JoeriTheGreat) [reply extensions/how-long-to-beat/src/index.tsx:56]: Still looking into the detail page. This is also broken.

Comment (Cronos87) [reply extensions/how-long-to-beat/src/index.tsx:56]: If you need/want, I can try to find time at noon to help you with this.

Comment (JoeriTheGreat) [reply extensions/how-long-to-beat/src/index.tsx:56]: The author stated the detail page is not working anymore https://github.com/ckatzorke/howlongtobeat/issues/40#issuecomment-1276691009
Only thing we can do is tigger the search instead and build the detail that way

Comment (pernielsentikaer) [reply extensions/how-long-to-beat/src/index.tsx:56]: We actually use prettier 2.8.8 🙂

**Comment by pernielsentikaer** [comment]
Is this ready to be merged?

**Comment by JoeriTheGreat** [comment]
> Is this ready to be merged?

i'm done and using the changes myself for the extension. 
Maybe @Cronos87 wants to do a code review and change things ?

**Comment by Cronos87** [comment]
I can do the code review in around 2 hours. Thank you @JoeriTheGreat!

File: extensions/how-long-to-beat/src/details.tsx [hunk 2]
```diff
@@ -1,10 +1,13 @@
...
-import { HowLongToBeatService, HowLongToBeatEntry } from "howlongtobeat";
+import { HowLongToBeatEntry } from "howlongtobeat";
 import { baseUrl } from ".";
 import { pluralize } from "./helpers";
+import axios from "axios";
+import UserAgent from "user-agents";
+import cheerio from "cheerio";
 
-const hltbService = new HowLongToBeatService();
+// const hltbService = new HowLongToBeatService();
```
Comment (Cronos87) [inline extensions/how-long-to-beat/src/details.tsx:10]: Don't hesitate to remove useless variables :)

File: extensions/how-long-to-beat/src/details.tsx [hunk 3]
```diff
@@ -96,3 +105,125 @@ ${result.playableOn.join(", ")}
...
+ * @param text
+ *            representing the hours
+ * @return the pares time as double
+ */
+function parseTime(text: string): number {
+  // '65&#189; Hours/Mins'; '--' if not known
+  if (text.startsWith("--")) {
+    return 0;
+  }
+  if (text.indexOf(" - ") > -1) {
```
Comment (Cronos87) [inline extensions/how-long-to-beat/src/details.tsx:200]:
```suggestion
  if (text.includes(" - ")) {
```

**Review by Cronos87 — APPROVED** [review]

**Comment by Cronos87** [comment]
Few comments, and it's ready to merge for me. Thank you again!

**Comment by pernielsentikaer** [comment]
@JoeriTheGreat Great! Thanks! 

Could you add a changelog entry, forgot to mention that earlier 🙂

**Comment by JoeriTheGreat** [comment]
> @JoeriTheGreat Great! Thanks!
> 
> Could you add a changelog entry, forgot to mention that earlier 🙂

Sure, no problem!

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Comment by Cronos87** [comment]
Thank you for the awesome work!

**Pull request merged**